src/dataExplorer/components/Results.tsx

```diff
@@ -25,31 +25,39 @@ import {FluxResult} from 'src/types/flows'
 
 import './Results.scss'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import {bytesFormatter} from 'src/shared/copy/notifications'
 
 // simplified version migrated from src/flows/pipes/Table/view.tsx
 const QueryStat: FC = () => {
   const {result} = useContext(ResultsContext)
 
   const tableColumn = result?.parsed?.table?.getColumn('table') || []
-  const lastTableValue = tableColumn[tableColumn.length - 1] || -1
+  const lastTableValue = tableColumn[tableColumn.length - 1]
```

Comment:
Good point! It was there because `tableColumn[tableColumn.length - 1]` can be undefined, which is the case before the "Run" button is clicked. I guess the logic `tableColumn[tableColumn.length - 1] || -1` didn't cover the case where the value of `tableColumn[tableColumn.length - 1]` is 0 and caused the bug. 

I saw Denise changed the condition check below from `else` to `else if (typeof lastTableValue === 'number')` to cover the undefined case. Nice!